docs/features-3.6.md

```diff
@@ -0,0 +1,77 @@
+# New features
+
+This is a concise list of new features.
+See [the upgrade notes](upgrading.md#upgrading_to_v3.6) for information on breaking changes and deprecations.
```

Comment:
```suggestion
For a more detailed list, see the [3.6 blog post](https://blog.argoproj.io/argo-workflows-3-6-aa037cd782be).

See [the upgrade notes](upgrading.md#upgrading_to_v3.6) for information on breaking changes and deprecations.
```

I mentioned this in the Contributor Meeting we talked about this, we should link to the blog post here as otherwise there are little to no links to the blog, making it hard to find.

Reply:
Done